test/utilsTests/did.testSuit.ts

```diff
@@ -1,8 +1,20 @@
-import { supportedDIDMethods } from '../../src/utils/did';
+import { supportedDIDMethods, parseDID } from '../../src/utils/did';
 import { Methods } from '@ew-did-registry/did';
 
 export function didTests() {
   it('Erc1056 method should be supported', () => {
     expect(supportedDIDMethods()).toContain(Methods.Erc1056);
   });
-}
+
+  it("should remove DID from address if present", async () => {
```

Comment:
As DID is did:method:address, I think more proper test name is "Should extract address from DID". Also `didAddress` is just `did` or `identity`